components/FieldGroup.tsx

```diff
@@ -27,7 +27,8 @@ const FieldGroup: FC<StyleProps> = ({ children, ...props }) => {
           sx: { ...styles?.child, ...child?.props?.sx },
           w:
             typeof child.type === 'function' &&
-            (child.type as FunctionComponent).displayName === 'TextField'
+            ((child.type as FunctionComponent).displayName === 'TextField' ||
+              (child.type as FunctionComponent).displayName === 'SelectField')
```

Comment:
Minor, but it might be nice to have some kind of utility for this?

```ts
const displayNameEquals = (x: ReactType, t: string) => (x.type as FunctionComponent).displayName === t
// [...]
displayNameEquals(child, 'TextField') ||
displayNameEquals(child, 'SelectField')
```